poll_suggestion/models.py

```diff
@@ -2,17 +2,20 @@
 from poll.models import Poll
 from django.db.utils import IntegrityError
 from users.models import Profile
+from django.utils import timezone
 
 
 class PollSuggestion(models.Model):
-    time = models.TimeField()
+    time = models.DateTimeField()
     poll_id = models.ForeignKey(Poll, on_delete=models.CASCADE)
 
     def save(self, *args, **kwargs):
         all_suggestions = PollSuggestion.objects.all()
         suggested_times = [poll_suggestion.time for poll_suggestion in all_suggestions]
         if self.time in suggested_times:
             raise IntegrityError
+        if self.time < timezone.now():
```

Comment:
in the pytest check in:
https://github.com/redhat-beyond/FitMeet/actions/runs/5014389530/jobs/8988572410?pr=104#step:6:169
you got the following error:
>       if self.time < timezone.now():
E       TypeError: can't compare offset-naive and offset-aware datetimes
because when you create the test object in:
https://github.com/redhat-beyond/FitMeet/pull/104/files#diff-7010e1edb5a6a9be6984e271fcfd634124788b9147d2c8ed37cab35b85890c0aR70
you create it like:
time=datetime.now() + timedelta(days=3)
without timezone
then here you try to compare it with an object that has a timezone which is ERROR
there are 2 options that could solve it:
1. use only datetime without timezone
2. use only datetime with timezone  
```suggestion
1.        if self.time < datetime.now():
OR
2.        if self.time < datetime.utcnow():
```

Reply:
I tried it in the first place with datetime but probably tried to make a few changes later. will revert to datetime.now() and check. thanks!